src/validation.cpp

```diff
@@ -739,7 +739,14 @@ bool MemPoolAccept::AcceptSingleTransaction(const CTransactionRef &ptx,
         return false;
     }
 
-    GetMainSignals().TransactionAddedToMempool(ptx);
+    std::vector<Coin> spent_coins;
+    spent_coins.reserve(ptx->vin.size());
```

Comment:
Why do you need `spent_coins` here? Avoid having to look them up via more calls?

Reply:
Yes, spent coins are not indexed. The data is right there in the node, so this simplifies the indexer.